newsroom/search.py

```diff
@@ -449,11 +450,21 @@ def prefill_search_products(self, search):
                     company_id=search.company.get("_id"),
                 )
             else:
-                search.products = get_products_by_company(
-                    search.company.get("_id"),
-                    search.navigation_ids,
-                    product_type=search.section,
-                )
+                user_products = []
+                if search.user and search.user.get("products"):
+                    user_products = get_products_by_user(
+                        search.user,
+                        search.section,
+                    )
+
+                if user_products:
```

Comment:
If a `User` has `Products` defined, and this is part of the functional spec, we have to keep in mind this when exposing it to the UI (and place it in the Newshub docs somewhere).

Otherwise I fear it may become confusing when a `Company` has a `Product` but `End User` cannot see the items because the `User` has a `Product` override in place

Reply:
yep right we will have to extend the current user management, in 2.4 this will be visible via company management which will be the only place where this could be actually set